.changeset/good-pumas-hunt.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+`TextField`: add support for `"month"`, `"week"`, and `"tel"` input types
```

Comment:
We aim to [use past tense in changelogs](https://github.com/hashicorp/design-system/blob/main/wiki/Website-Changelog.md#tips-for-writing-consistent-changelog-entries) – and yes, I see how that's unnatural compared to git commit messages

```suggestion
`Form::TextInput` - added support for `"month"`, `"week"`, and `"tel"` input types
```